File: apzkr-pzpi-21-5-vyshniak-viktoriia/Task1-Server/BLL/Infrastructure/Models/User/UserProfileModel.cs
using Common.Enums;

namespace BLL.Infrastructure.Models.User;
public class UserProfileModel
{
    public int UserId { get; set; }

    public Region Region { get; set; }

    public Gender Gender { get; set; }

    public string Email { get; set; }

    public string Login { get; set; }

    public DateTime RegisteredOnUtc { get; set; }

    public Role Role { get; set; }
}

File: apzkr-pzpi-21-5-vyshniak-viktoriia/Task3-WebClient/src/app/core/models/user/user-profile-model.ts
import { Gender } from "../../enums/gender";
import { Region } from "../../enums/region";
import { Role } from "../../enums/role";

export interface UserProfileModel {
	userId: number;
	email: string;
	login: string;
	role: Role;
	gender: Gender,
	region: Region,
	registeredOnUtc: Date
}
